src/Microsoft.DotNet.Interactive.ExtensionLab.Tests/MermaidKernelExtensionTests.cs

```diff
@@ -70,6 +70,23 @@ graph TD
             this.Assent(formatted.FixedGuid());
         }
 
+        [Fact]
+        public async Task registers_html_formatter_for_explorer()
+        {
+            using CompositeKernel kernel = new CompositeKernel
+            {
+                new CSharpKernel().UseNugetDirective(),
+            };
+
+            var extension = new MermaidKernelExtension();
+
+            await extension.OnLoadAsync(kernel);
+
+            string formatted = typeof(List<string>).ExploreWithUmlClassDiagram().ToDisplayString(HtmlFormatter.MimeType);
+
+            this.Assent(formatted.FixedGuid());
```

Comment:
This test also doesn't really verify what the name of the test is verifying. It's verifying that the output matches the approval, which isn't readable from here. It's not verifying the content. For example, any object can be formatted to HTML using our default formatters. We should probably verify something about the content itself.

Reply:
Ok will fix it later

Reply:
changed